### DISCUSSION TITLE
Centrar ul/il

### DISCUSSION THREAD

**Discussion by ascamarena** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#177] [Q&A] [answered]
Hola Explorers, espero les esté yendo muy bien.

Mi duda es la siguiente:
Quiero centrar los "puntitos" que aparecen en las listas. Este es mi código:
![image](https://user-images.githubusercontent.com/92229619/156231213-aef718b3-6eac-4787-9ef5-477797f84a90.png)

Pero sale así:

![image](https://user-images.githubusercontent.com/92229619/156231394-204b4bff-ca32-4701-b760-3a7db1a7f9b4.png)

¿Alguna idea?
Gracias de antemano 🎉

PD: Leí que se podía hacer pero en CSS y me gustaría saber si puedo hacerlo en HTML.
PD2: Pueden intentar `<ul style="display: inline-block"></ul/>` {También funciona con `<il>`}

**Comment by AngelCruzO** [comment]
Tendrías que utilizar la propiedad aling-text o text-aling en el ul con el valor Center. Pero es mejor con css

**Comment by doguedogue** [comment]
Puedes usar una tabla

**Reply by OmarVenturaP** [reply]
A mi me pasa lo mismo, la verdad estoy viendo la posibilidad de cambiar mi código y agregarle una tabla.

**Comment by Ingrzs** [comment, marked as answer]
Bien puede ser en css
Asi
ul o li {
Align-items;center;
}

O con justify-content
